### ISSUE TITLE
Error in invoker

### ISSUE THREAD

**Issue by tobya** [beyondcode/invoker-community#342]
------
**OS**: win32
**Invoker Version**: 2.11.0
**Laravel Version**: 10.11.0
**Local project**: unknown
**PHP Binary**: C:\Development\php\php81\php.exe
**Route**: /project/index?class=App%5CEmailAutomation&time=1685086176163&project=remote-rBtU8K1da1UGuuFPWJ2nLE

**Error**:
```
Error 

  Call to undefined method Doctrine\DBAL\Connection::createSchemaManager()

  at backoffice.cookingisfun.ie/vendor/laravel/framework/src/Illuminate/Database/Connection.php:1145
    1141▕     public function getDoctrineSchemaManager()
    1142▕     {
    1143▕         $connection = $this->getDoctrineConnection();
    1144▕ 
  ➜ 1145▕         return $connection->createSchemaManager();
    1146▕     }
    1147▕ 
    1148▕     /**
    1149▕      * Get the Doctrine DBAL database connection instance.

  1   phar:///tmp/forge_invoker.phar/src/Factories/AttributeCollection.php:25
      Illuminate\Database\Connection::getDoctrineSchemaManager()

  2   phar:///tmp/forge_invoker.phar/src/Factories/AttributeCollection.php:15
      _HumbugBox8e7f5be646a5\Invoker\Factories\AttributeCollection::getColumns()
```

**Comment by mechelon** [comment]
We released version 2.13.0 recently - could you please try this version? If you still get the error, can you share the content of your `composer.json` file?

**Comment by tobya** [comment]
Still happening with latest version

![image](https://github.com/beyondcode/invoker-community/assets/325502/893e3974-7b10-4c59-9ab8-f209fb023c34)

here is the composer.json

````
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": [
        "framework",
        "laravel"
    ],
    "license": "MIT",
    "require": {
        "php": "^8.1",
        "arcanedev/log-viewer": "^10.0.0",
        "backpack/crud": "^5.5.5",
        "backpack/permissionmanager": "^6.0.1",
        "bensampo/laravel-enum": "^6.0.0",
        "brick/phonenumber": "^0.4.0",
        "bugsnag/bugsnag-laravel": "^2.0",
        "calebporzio/sushi": "^2.3",
        "chillerlan/php-qrcode": "^4.3",
        "components/font-awesome": "^5.15",
        "divineomega/php-countries": "^2.3",
        "expertlead/webflow-php-sdk": "^0.1.1",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^10.0.0",
        "laravel/sanctum": "^3.2",
        "laravel/tinker": "^2.0",
        "livewire/livewire": "^2.4",
        "maatwebsite/excel": "^3.1",
        "predis/predis": "^1.1",
        "smarty/smarty": "^4.2",
        "spatie/laravel-activitylog": "^4.6.0",
        "spatie/laravel-medialibrary": "^10.0.0",
        "spatie/laravel-permission": "^5.5.5",
        "spatie/laravel-tags": "^4.0.0",
        "spatie/once": "^3.1",
        "spatie/url": "^1.3",
        "stripe/stripe-php": "^7.66",
        "sven/artisan-view": "^3.5.0",
        "telegram-bot/api": "^2.3",
        "tobya/anymodel": "^1.5.1",
        "tobya/bcsapiwrapper": "^4.10",
        "tobya/bcsmodel": "^1.1",
        "tobya/devroute": "^1.1",
        "tobya/lassi": "dev-main",
        "tobya/makepest": "^0.5.0",
        "tobya/pprint_r": "^2.1",
        "tobya/queuestatus": "dev-main",
        "tobya/summarychart": "dev-master"
    },
    "require-dev": {
        "backpack/generators": "^3.1",
        "barryvdh/laravel-debugbar": "^3.6",
        "fakerphp/faker": "^1.21",
        "kitloong/laravel-migrations-generator": "^6.4.2",
        "laravel/breeze": "^1.3",
        "laravel/sail": "^1.10",
        "mockery/mockery": "^1.3.1",
        "nunomaduro/collision": "^7.0",
        "nunomaduro/larastan": "^2.0.0",
        "pestphp/pest": "^2.0",
        "phpunit/phpunit": "^10.0",
        "spatie/laravel-ignition": "^2.0"
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": {
            "*" : "dist"
        },
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true
        }
    },
    "extra": {
        "laravel": {
            "dont-discover": [

            ]
        }
    },
    "autoload": {
             "files": [
            "app/Helpers/helpers.php"
        ],
        "psr-4": {
            "App\\": "app/",
             "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/",
            "BCS\\Example\\SubNamespace\\" : "Modules/SubNamespace/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true,
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi",
            "@php artisan vendor:publish --force --tag=livewire:assets --ansi"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ],
        "post-update-cmd": [
            "@php artisan bcs:checksqldate --update"
        ],
        "post-install-cmd": [
            "@php artisan bcs:checksqldate --update"
        ]


    }
}

````

**Comment by tobya** [comment]
Hello any ideas on this.  It prevents me from opening any model on my project.

**Comment by mechelon** [comment]
Can you try the following:
1. Close Invoker
2. Delete the `/tmp/forge_invoker.phar` file from the server
3. Open Invoker again & reconnect to the project.

**Comment by tobya** [comment]
File doesnt seem to exist.

I'm running on Windows.  

I checked 

> C:\Users\tobyl\AppData\Local\Temp

and ran a search but no .phar files.

I found invoker.phar in the root app directory, but nothing else.

Any other places it might be on windows?

**Comment by mechelon** [comment]
Yes, these are local files - but your initial issue description seems like you were connected to a remote server?

**Comment by domthomas-dev** [comment]
I have the same issue. I delete `/tmp/forge_10_invoker.phar` on the server, but always the same message.

------
**OS**: darwin
**Invoker Version**: 2.13.0
**Laravel Version**: 10.6.2
**Local project**: unknown
**PHP Binary**: /usr/local/bin/php
**Route**: /project/index?class=App%5CModels%5CActivity&time=1691746449122&project=remote-4orbhbjUSF8vMYjVwvgU8t

**Error**:
```
Error 

  Call to undefined method Doctrine\DBAL\Connection::createSchemaManager()

  at pim-prod.arstime.tech/vendor/laravel/framework/src/Illuminate/Database/Connection.php:1145
    1141▕     public function getDoctrineSchemaManager()
    1142▕     {
    1143▕         $connection = $this->getDoctrineConnection();
    1144▕ 
  ➜ 1145▕         return $connection->createSchemaManager();
    1146▕     }
    1147▕ 
    1148▕     /**
    1149▕      * Get the Doctrine DBAL database connection instance.

  1   phar:///tmp/forge_10_invoker.phar/src/Factories/AttributeCollection.php:25
      Illuminate\Database\Connection::getDoctrineSchemaManager()

  2   phar:///tmp/forge_10_invoker.phar/src/Factories/AttributeCollection.php:15
      _HumbugBox0aaa99c27ffd\Invoker\Factories\AttributeCollection::getColumns()
```

**Comment by mechelon** [comment]
@domthomas-dev  - [ ] 13:00 #Studentenstatus

**Comment by mechelon** [comment]
@domthomas-dev Which PHP version do you use?

**Comment by mechelon** [comment]
We just released Invoker 2.13.1 which should fix the problem.

**Comment by tobya** [comment]
That seems to have fixed it. Thanks
